Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+   "metadata": {},
+   "source": [
+    "# A little statistis Q1B\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 143,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "from scipy import stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 144,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8413447460685429\n",
+      "0.9986501019683699\n",
+      "0.0013498980316301035\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Integrate the standard normal distribution\n",
+    "prob1 = stats.norm.cdf(1)\n",
+    "prob2 = stats.norm.cdf(3)\n",
+    "prob3 = 1- stats.norm.cdf(3)\n",
+    "\n",
+    "print(prob1)\n",
+    "print(prob2)\n",
+    "print(prob3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The probabilities match the cumulative z values.\n",
+    "Z = 1 => 0.84134\n",
+    "Z = 3 => 0.99865\n",
+    "Z = -3 => 0.00003\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q1C"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 145,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.0\n",
+      "2.9999999999999982\n",
+      "-2.9999999999999982\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Calculate the sigma\n",
+    "sigma1 = stats.norm.ppf(0.8413447460685429)\n",
+    "sigma2 = stats.norm.ppf(0.9986501019683699)\n",
+    "sigma3 = stats.norm.ppf(0.0013498980316301035)\n",
+    "\n",
+    "print(sigma1)\n",
+    "print(sigma2)\n",
+    "print(sigma3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We know from Q1B the correct sigma asoociated with each probability and the sigma values match that values."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q1D\n",
+    "We can see in Q1B that norm.cdf() integrates the standard normal distribution from minus infinity to the sigma value given as the parameter. Generally, when we are looking for a probability associated with a specific sigma value, we are looking for a probility to the right of the sigma value, i.e. from that sigma value to positive infinity. So in this case, we will need to calculate the probability using either 1-norm.cdf(positive sigma) or norm.cdf(minus sigma). Since normal distribution is symmetrical, integration the distribution from minus infinity to minius sigma value will return the correct tail-end probability. So depnding on the context, we will need to use positive sigma or negative sigma to get the correct probability."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q2A"
```

Comment:
I think it will be nice if you can mention which distribution you chose for this part of the lab.

Reply:
Thanks it seems like I copy pasted the wrong value.